src/entities/diffs/lib/pullRequestFileQuerier.ts

```diff
@@ -0,0 +1,38 @@
+import { PullRequestFile } from '../../pullRequestFiles/model';
+import { PullRequest } from '../../pullRequests/model/schemas';
+import { OCTOKIT } from '../../../lib';
+import { RepositoryIdentifier } from '../../repositories/model';
+
+export async function getPullRequestFiles(
+  repoIdent: RepositoryIdentifier,
+  pullRequest: PullRequest,
+) {
+  const pullRequestFiles: PullRequestFile[] = [];
+  let page: PullRequestFile[] = [];
+  let pageNumber = 1;
+
+  do {
+    page = await queryPullRequestFilePage(repoIdent, pullRequest, pageNumber);
+    pullRequestFiles.push(...page);
```

Comment:
I believe we need to use the generator pattern here as we do in the pullRequestQuerier. Else this array will easily overflow